File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/1182.cs
using System;

class URI
{
    static void Main() {
        int l, i, j;
        double valor, soma=0;
        string escolha;
        l = int.Parse(System.Console.ReadLine().Trim());
        escolha = System.Console.ReadLine().Trim();
        for(i=0;i<12;i++){
            for(j=0;j<12;j++){
                valor = double.Parse(System.Console.ReadLine().Trim());
                if(j==l){
                    soma+=valor;
                }
            }
        }
        
        if(escolha=="S"){
            Console.Write("{0:f1}\n", soma);
        } else{
            Console.Write("{0:f1}\n", soma/12);
        }
    }
}

File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/1182.java
import java.util.Scanner;

public class Main {
    public static void main(String args[]) {
        Scanner ler = new Scanner(System.in);
        int l, i, j;
        double valor, soma=0;
        String escolha;
        l = ler.nextInt();
        escolha = ler.next();
        for(i=0;i<12;i++){
            for(j=0;j<12;j++){
                valor = ler.nextDouble();
                if(j==l){
                    soma+=valor;
                }
            }
        }
        
        if(escolha.equals('S')){
            System.out.printf("%.1f\n", soma);
        } else{
            System.out.printf("%.1f\n", soma/12);
        }

    }
}